features/git-town/configuration/configuration_initial.feature

```diff
@@ -0,0 +1,50 @@
+Feature: Initial configuration
+
+  As a user who hasn't configured Git Town yet
+  I want to have a simple, dedicated setup command
+  So that I can configure it safely before using any Git Town command
+
+
+  Background:
+    Given I haven't configured Git Town yet
+
+
+  Scenario: user enters non-existent main branch
+    When I run `git town config --setup` and enter "nonexistent"
+    Then I get the error
+      """
+        Error
+        There is no branch named 'nonexistent'
+      """
+    And Git Town is still not configured for this repository
+
+
+  Scenario: user enters a valid main branch and non-existent non-feature branch
+    Given I have a branch named "master"
+    When I run `git town config --setup` and enter "master" and "nonexistent"
+    Then I get the error
+      """
+        Error
+        There is no branch named 'nonexistent'
+      """
+    And the main branch name is now configured as "master"
+    And my non-feature branches are still not configured
+
+
+  Scenario: user enters valid main branch and valid non-feature branches
+    Given I have branches named "dev" and "qa"
+    When I run `git town config --setup` and enter "dev" and "qa"
```

Comment:
How do you feel about making it more explicit which input belongs to which step? 

```cucumber
When I run `git town config --setup` and enter the main branch name "dev" and the non-feature branches "qa" 
```

This would also allow to change the interface of the config setup command, or the order in which branch names are queried, without having to make a shotgun refactoring of all these scenarios.